components/description-list/description-list-wrapper.js

```diff
@@ -0,0 +1,100 @@
+import { bodyCompactStylize, labelStylize } from '../typography/styles.js';
+import { css, html, LitElement } from 'lit';
+import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
+
+export const descriptionListStyles = [
+	labelStylize(css`dt`),
+	bodyCompactStylize(css`dd`),
+	css`
+		:host {
+			--d2l-dt-min-width: min-content;
+			--d2l-dt-max-width: 10rem;
+			--d2l-dd-min-width: 50%;
+		}
+		dl {
+			display: grid;
+			grid-auto-flow: row;
+			align-items: baseline;
+			grid-template-columns: minmax(var(--d2l-dt-min-width), auto) minmax(var(--d2l-dd-min-width), 1fr);
+			gap: 0.5rem 1.5rem;
+		}
+		dt {
+			max-width: var(--d2l-dt-max-width);
+		}
+		dd {
+			margin: 0;
+		}
+		d2l-dl-wrapper[stacked] dl {
+			display: block;
+		}
+		d2l-dl-wrapper[stacked] dt {
+			max-width: unset;
+		}
+		d2l-dl-wrapper[stacked] dd {
+			margin-bottom: 0.5rem;
+		}
+	`
+];
+
+/**
+ * Wraps a native <dl> element, providing styling and resize behavior.
+ * @slot - Content to wrap
+ */
+class DescriptionListWrapper extends LitElement {
+	static get properties() {
+		return {
+			stacked: { type: Boolean, reflect: true },
+			breakpoint: { type: Number, reflect: true },
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.stacked = false;
+		this.breakpoint = 350;
+		this._resizeObserver = new ResizeObserver(this._onResize.bind(this));
+		this._checkIfShouldStack();
```

Comment:
I wouldn't call this from the constructor, since it won't be attached to the DOM yet -- do this in `connectedCallback` or `firstUpdated`, or if the resize observer always fires when `observe()` is first called (I forget if that's true), then you may not even need to call it.

Reply:
I'm not sure what you mean by the following:

> if the resize observer always fires when observe() is first called

Reply:
Like in `connectedCallback()` you're calling `.observe()` on it -- if that immediately triggers the callback, then you don't need to also call `_checkIfShouldStack()` when initializing since it'll do that for you. But I don't remember if that happens initially or if it only fires when a future-resize event happens.

Reply:
Gotcha. It does look like it fires initially, so I'll remove it from the constructor.